module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraApplicationPolicy.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Remove-EntraApplicationPolicy
+description: This article provides details on the Remove-EntraApplicationPolicy command.
+
+
+ms.topic: reference
+ms.date: 06/6/2024
+ms.author: eunicewaweru
+ms.reviewer: stevemutungi
+manager: CelesteDG
+author: msewaweru
+
+external help file: Microsoft.Graph.Entra-Help.xml
+Module Name: Microsoft.Graph.Entra
+online version: https://learn.microsoft.com/powershell/module/Microsoft.Graph.Entra/Remove-EntraScopedRoleMembership
+
+schema: 2.0.0
+---
+
+# Remove-EntraApplicationPolicy
+
+## Synopsis
+
+Removes an application policy.
+
+## Syntax
+
+```powershell
+Remove-EntraApplicationPolicy 
+ -Id <String> 
+ -PolicyId <String> 
+ [<CommonParameters>]
+```
+
+## Description
+
+The `Remove-EntraApplicationPolicy` cmdlet removes an application policy from Microsoft Entra ID.
+
+## Examples
+
+### Example 1: Remove an application policy
+
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All'
+$params = @{
+    Id = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
+    PolicyId = 'bbbbbbbb-1111-2222-3333-cccccccccccc'
+}
+Remove-EntraApplicationPolicy @params
+```
+
+This command removes the specified application policy.
+
+## Parameters
+
+### -PolicyId
+
+Specifies the ID of the policy.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -Id
+
+Specifies the ID of Policy.
```

Comment:
The `PolicyId` and `Id` have the same description. Confirm if this should be the case.

Reply:
This cmdlet is not working i accidentally added that document.